torch/optim/adadelta.py

```diff
@@ -63,7 +81,9 @@ def _init_group(self, group, params_with_grad, grads, square_avgs, acc_deltas):
 
             # Lazy state initialization
             if len(state) == 0:
-                state["step"] = 0
+                state["step"] = torch.zeros(
+                    (), device=p.device, dtype=_get_scalar_dtype()
```

Comment:
We don't want this on p's device by default on eager

Reply:
oh, should I add a capturable arg here then?